src/main/java/io/github/darealturtywurty/ancientology/core/init/BlockInit.java

```diff
@@ -1,41 +1,96 @@
 package io.github.darealturtywurty.ancientology.core.init;
 
-import io.github.darealturtywurty.ancientology.Ancientology;
-import io.github.darealturtywurty.ancientology.core.util.LootTableUtils;
-import io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder;
-import io.github.darealturtywurty.ancientology.core.util.registry.BlockDeferredRegister;
-import io.github.darealturtywurty.ancientology.core.util.registry.BlockRegistryObject;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.HarvestLevel.STONE;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.HarvestLevel.WOOD;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.HarvestTool.AXE;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.HarvestTool.HOE;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.HarvestTool.PICKAXE;
+import static io.github.darealturtywurty.ancientology.core.util.registry.BlockBuilder.RenderLayer.CUTOUT;
+
+import java.util.Random;
+
+import net.minecraft.tags.BlockTags;
 import net.minecraft.util.valueproviders.UniformInt;
+import net.minecraft.world.item.Items;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.LeavesBlock;
 import net.minecraft.world.level.block.OreBlock;
+import net.minecraft.world.level.block.RotatedPillarBlock;
+import net.minecraft.world.level.block.SaplingBlock;
+import net.minecraft.world.level.block.grower.AbstractTreeGrower;
+import net.minecraft.world.level.levelgen.feature.ConfiguredFeature;
+
+import io.github.darealturtywurty.ancientology.Ancientology;
+import io.github.darealturtywurty.ancientology.core.util.LootTableUtils;
+import io.github.darealturtywurty.ancientology.core.util.MinecraftLocale;
+import io.github.darealturtywurty.ancientology.core.util.registry.BlockDeferredRegister;
+import io.github.darealturtywurty.ancientology.core.util.registry.BlockRegistryObject;
+import io.github.darealturtywurty.ancientology.core.worldgen.FeatureGen;
 import net.minecraftforge.common.Tags;
 
+//@formatter:off
```

Comment:
I don't see why you turned the formatter off. I believe you can remove this.

Reply:
I turned it off because using the formatter there will mess up all the nice method chaining

Reply:
Well then it should be re-enabled afterwards.

Reply:
There's nothing afterwards, only the constrctor

Reply:
and?